tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -640,6 +626,51 @@ def _run(self):
                         pcnt_used=0
                     ).save()
 
+                # Invoke the aggregated sync flow
+                if "provisioner/gluster" in NS.node_context.tags:
+                    params = {}
+                    params['TendrlContext.integration_id'] = \
+                        NS.tendrl_context.integration_id
+                    payload = {
+                        "tags": ["provisioner/gluster"],
+                        "run": "gluster.flows.SyncAggregatedClusterDetails",
+                        "status": "new",
+                        "parameters": params,
+                        "type": "sds"
+                    }
+                    _job_id = str(uuid.uuid4())
+                    Job(
+                        job_id=_job_id,
+                        status="new",
+                        payload=payload
+                    ).save()
+                    Event(
+                        Message(
+                            priority="info",
+                            publisher=NS.publisher_id,
+                            payload={
+                                "message": "Started job %s for aggregated sync"
+                                           "of cluster %s" %
+                                (_job_id, NS.tendrl_context.integration_id)
+                            }
+                        )
+                    )
+                    while True:
```

Comment:
should we worry about the result of the job ? can the job itself log the failure if any? thus sync is finished faster ?
anyhow, this job is not something fired by user, so its result is not as important as a user fired job.

just a thought.